ERROR - 2024-06-19 00:35:28 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`almacen`.`producto`, CONSTRAINT `producto_ibfk_2` FOREIGN KEY (`idproveedor`) REFERENCES `proveedor` (`idproveedor`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE `product...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `product...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `product...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `product...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(854): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(126): CodeIgniter\Model->update('12', Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->update('12')
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 00:35:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`almacen`.`producto`, CONSTRAINT `producto_ibfk_2` FOREIGN KEY (`idproveedor`) REFERENCES `proveedor` (`idproveedor`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: PUT, Route: productos/12]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `producto` SET `nombreproducto` = :nombreproducto:, `precioventa` = :precioventa:, `ubicacionproducto` = :ubicacionproducto:, `codigobarras` = :codigobarras:, `idcategoria` = :idcategoria:, `idproveedor` = :idproveedor:
WHERE `producto`.`idproducto` IN :producto.idproducto:', [...], false)
 2 SYSTEMPATH\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Productos.php(126): CodeIgniter\Model->update('12', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->update('12')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 00:35:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`almacen`.`producto`, CONSTRAINT `producto_ibfk_2` FOREIGN KEY (`idproveedor`) REFERENCES `proveedor` (`idproveedor`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `producto` SET `nombreproducto` = \'papaya\', `precioventa` = \'1\', `ubicacionproducto` = \'no me acuerdo jiidddd\', `codigobarras` = \'1234\', `idcategoria` = \'29\', `idproveedor` = \'20\'
WHERE `producto`.`idproducto` IN (\'12\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `producto` SET `nombreproducto` = \'papaya\', `precioventa` = \'1\', `ubicacionproducto` = \'no me acuerdo jiidddd\', `codigobarras` = \'1234\', `idcategoria` = \'29\', `idproveedor` = \'20\'
WHERE `producto`.`idproducto` IN (\'12\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `producto` SET `nombreproducto` = :nombreproducto:, `precioventa` = :precioventa:, `ubicacionproducto` = :ubicacionproducto:, `codigobarras` = :codigobarras:, `idcategoria` = :idcategoria:, `idproveedor` = :idproveedor:
WHERE `producto`.`idproducto` IN :producto.idproducto:', [...], false)
 4 SYSTEMPATH\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Productos.php(126): CodeIgniter\Model->update('12', [...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->update('12')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 00:35:28 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`almacen`.`producto`, CONSTRAINT `producto_ibfk_2` FOREIGN KEY (`idproveedor`) REFERENCES `proveedor` (`idproveedor`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE `producto` SET `nombreproducto` = \'papaya\', `precioventa` = \'1\', `ubicacionproducto` = \'no me acuerdo jiidddd\', `codigobarras` = \'1234\', `idcategoria` = \'29\', `idproveedor` = \'20\'
WHERE `producto`.`idproducto` IN (\'12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `producto` SET `nombreproducto` = \'papaya\', `precioventa` = \'1\', `ubicacionproducto` = \'no me acuerdo jiidddd\', `codigobarras` = \'1234\', `idcategoria` = \'29\', `idproveedor` = \'20\'
WHERE `producto`.`idproducto` IN (\'12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `producto` SET `nombreproducto` = \'papaya\', `precioventa` = \'1\', `ubicacionproducto` = \'no me acuerdo jiidddd\', `codigobarras` = \'1234\', `idcategoria` = \'29\', `idproveedor` = \'20\'
WHERE `producto`.`idproducto` IN (\'12\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2497): CodeIgniter\Database\BaseConnection->query('UPDATE `producto` SET `nombreproducto` = :nombreproducto:, `precioventa` = :precioventa:, `ubicacionproducto` = :ubicacionproducto:, `codigobarras` = :codigobarras:, `idcategoria` = :idcategoria:, `idproveedor` = :idproveedor:
WHERE `producto`.`idproducto` IN :producto.idproducto:', [...], false)
 5 SYSTEMPATH\Model.php(450): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(854): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Productos.php(126): CodeIgniter\Model->update('12', [...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->update('12')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 01:44:16 --> Error: Call to undefined method App\Controllers\Proveedor::setResponseFormat()
[Method: GET, Route: proveedor]
in APPPATH\Controllers\Proveedor.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Proveedor->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Proveedor))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 01:48:06 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 01:48:06 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 01:48:06 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'g\', \'rreer\', \'eer\', \'erere\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'g\', \'rreer\', \'eer\', \'erere\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 01:48:06 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'g\', \'rreer\', \'eer\', \'erere\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'g\', \'rreer\', \'eer\', \'erere\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'g\', \'rreer\', \'eer\', \'erere\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 01:48:43 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 01:48:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 01:48:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fds\', \'sdf\', \'dsf\', \'sdf\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fds\', \'sdf\', \'dsf\', \'sdf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 01:48:43 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fds\', \'sdf\', \'dsf\', \'sdf\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fds\', \'sdf\', \'dsf\', \'sdf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fds\', \'sdf\', \'dsf\', \'sdf\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 01:53:17 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 01:53:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 01:53:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'qwer\', \'sdf\', \'sdf\', \'sadf\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'qwer\', \'sdf\', \'sdf\', \'sadf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 01:53:17 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'qwer\', \'sdf\', \'sdf\', \'sadf\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'qwer\', \'sdf\', \'sdf\', \'sadf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'qwer\', \'sdf\', \'sdf\', \'sadf\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 02:00:12 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 02:00:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:00:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfg\', \'cv\', \'df\', \'df\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfg\', \'cv\', \'df\', \'df\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:00:12 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfg\', \'cv\', \'df\', \'df\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfg\', \'cv\', \'df\', \'df\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfg\', \'cv\', \'df\', \'df\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 02:04:52 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 02:04:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:04:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfgh\', \'fgh\', \'fgh\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfgh\', \'fgh\', \'fgh\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:04:52 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfgh\', \'fgh\', \'fgh\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfgh\', \'fgh\', \'fgh\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfgh\', \'fgh\', \'fgh\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 02:05:05 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 02:05:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:05:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfg\', \'fgh\', \'vb\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfg\', \'fgh\', \'vb\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:05:05 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfg\', \'fgh\', \'vb\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfg\', \'fgh\', \'vb\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'fgh\', \'dfg\', \'fgh\', \'vb\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 02:06:51 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 02:06:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:06:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'rdtfy\', \'fdg\', \'fdg\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'rdtfy\', \'fdg\', \'fdg\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:06:51 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'rdtfy\', \'fdg\', \'fdg\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'rdtfy\', \'fdg\', \'fdg\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'rdtfy\', \'fdg\', \'fdg\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 02:08:52 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 02:08:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:08:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfs\', \'dasf\', \'adsf\', \'asdf\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfs\', \'dasf\', \'adsf\', \'asdf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:08:52 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfs\', \'dasf\', \'adsf\', \'asdf\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfs\', \'dasf\', \'adsf\', \'asdf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'dfs\', \'dasf\', \'adsf\', \'asdf\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 02:12:58 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 02:12:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:12:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdfg\', \'szdxfcg\', \'szdxfcg\', \'zxcv\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdfg\', \'szdxfcg\', \'szdxfcg\', \'zxcv\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:12:58 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdfg\', \'szdxfcg\', \'szdxfcg\', \'zxcv\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdfg\', \'szdxfcg\', \'szdxfcg\', \'zxcv\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdfg\', \'szdxfcg\', \'szdxfcg\', \'zxcv\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 02:13:22 --> mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Productos.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Productos->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 02:13:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
[Method: POST, Route: productos]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:13:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'sdf\', \'sdf\', \'sdf\', \'13\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'sdf\', \'sdf\', \'sdf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 02:13:22 --> [Caused by] mysqli_sql_exception: Duplicate entry '13' for key 'idcategoria'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'sdf\', \'sdf\', \'sdf\', \'13\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'sdf\', \'sdf\', \'sdf\', \'13\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (\'sdf\', \'sdf\', \'sdf\', \'sdf\', \'13\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `producto` (`nombreproducto`, `precioventa`, `ubicacionproducto`, `codigobarras`, `idcategoria`, `idproveedor`) VALUES (:nombreproducto:, :precioventa:, :ubicacionproducto:, :codigobarras:, :idcategoria:, :idproveedor:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Productos.php(80): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Productos->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-19 03:10:09 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `ve...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ve...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ve...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ve...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Ventas.php(78): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Ventas->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-19 03:10:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
[Method: POST, Route: ventas]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (:dnicliente:, :nombrecliente:, :fechaventa:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ventas.php(78): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 03:10:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (\'d\', \'d\', \'df\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (\'d\', \'d\', \'df\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (:dnicliente:, :nombrecliente:, :fechaventa:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ventas.php(78): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-19 03:10:09 --> [Caused by] mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (\'d\', \'d\', \'df\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (\'d\', \'d\', \'df\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (\'d\', \'d\', \'df\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `venta` (`dnicliente`, `nombrecliente`, `fechaventa`) VALUES (:dnicliente:, :nombrecliente:, :fechaventa:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ventas.php(78): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
